examples/webgl_loader_texture_basis.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - basis texture loader</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+		<style>
+			body {
+				margin: 0px;
+				background-color: #050505;
+				color: #fff;
+				overflow: hidden;
+			}
+
+			a { color: #e00 }
+
+			#info {
+				position: absolute;
+				top: 0px;
+				width: 100%;
+				color: #ffffff;
+				padding: 5px;
+				font-family: Monospace;
+				font-size: 13px;
+				text-align: center;
+				z-index: 1000;
+			}
+		</style>
+	</head>
+	<body>
+
+		<div id="info">
+			<a href="http://threejs.org" target="_blank" rel="noopener">three.js</a> - webgl - basis texture loader<br />
+			<a href="https://github.com/binomialLLC/basis_universal" target="_blank">Basis Universal GPU Texture Codec</a>
+		</div>
+
+		<script src="../build/three.js"></script>
+		<script src="js/loaders/BasisTextureLoader.js"></script>
+
+		<script>
+
+			var camera, scene, renderer;
+			var mesh;
+
+			var clock = new THREE.Clock();
+
+			init();
+			animate();
+
+			function init() {
+
+				renderer = new THREE.WebGLRenderer( { antialias: true } );
+				renderer.setPixelRatio( window.devicePixelRatio );
+				renderer.setSize( window.innerWidth, window.innerHeight );
+				document.body.appendChild( renderer.domElement );
```

Comment:
BasisTextureLoader currently marks all textures as sRGB:

https://github.com/mrdoob/three.js/blob/b39de785004e155e744fc47604de5e973135746b/examples/js/loaders/BasisTextureLoader.js#L144-L151

We should probably remove that, and the `flipY=false` preset, which both made sense in the original Basis example but not so much in a general-purpose texture loader.

My suggestion for this PR would be:

- [ ] Remove flipY=false and encoding=sRGBEncoding from the loader
- [ ] Add encoding=sRGBEncoding after loading the texture (in the example itself)
- [ ] set renderer.gammaOutput=true, renderer.gammaFactor=2.2 as best practice.

Reply:
> Remove flipY = false from the loader

`flipY = true` doesn't seem to work for compressed texture and default `CompressedTexture.flipY` is `false`. Removing from the loader looks right direction. Updated.

> Move encoding = sRGBEncoding into example

Updated.

> renderer.gammaOutput = true, renderer.gammaFactor=2.2

Thanks, I missed them. Updated.